msteams-platform/includes/bots/user-mention.md

```diff
@@ -0,0 +1,10 @@
+---
+title: User Mention
+description: With this learning module, learn how your bot can mention other users in messages posted in channels.
+localization_priority: Normal
+ms.topic: overview
+ms.author: surbhigupta12
+ms.date: 11/14/2024
+---
+
+Bots with Adaptive Cards support user mention IDs, such as Microsoft Entra Object ID and User Principle Name (UPN) in addition to the existing IDs. Incoming Webhooks start to support user mention in Adaptive Card with the Microsoft Entra Object ID and UPN.
```

Comment:
Please remove "with Adaptive Cards". It should be: "Bots support user mention IDs, such as Microsoft.....".

Reply:
@limamicro I've updated the input.